src/main/java/org/folio/search/service/converter/ElasticsearchFacetConverter.java

```diff
@@ -76,4 +83,23 @@ private static FacetItem facetItem(Bucket bucket) {
   private static FacetResult facetResult(Map<String, Facet> facets) {
     return new FacetResult().facets(facets).totalRecords(facets.size());
   }
+
+  private static Map<String, Facet> mergeSelectedAndNormalFacets(Map<String, Facet> facets) {
+    var result = new LinkedHashMap<String, Facet>();
+
+    facets.forEach((key, value) -> {
+      var isSelectedAggregation = isSelectedTermsAggregation(key);
+      var finalKey = isSelectedAggregation ? key.substring(SELECTED_AGG_PREFIX.length()) : key;
```

Comment:
Are we adding this synthetic prefix to distinguish selected vs all facets?

Reply:
yes, it is common practice to segregate aggregation values (also, Elasticsearch doesn't allow to user to have similar aggregation names at the same level)